scrapers/FemdomEmpire.yml

```diff
@@ -0,0 +1,63 @@
+name: "FemdomEmpire"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - femdomempire.com/tour/trailers/
+      - feminized.com/tour/trailers/
+    scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - femdomempire.com/tour/models/
+      - feminized.com/tour/models/
+    scraper: performerScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="videoDetails clear"]/h3
+      Date:
+        selector: //p[span[contains(.,"Date Added")]]
+        postProcess:
+          - replace:
+            - regex: "Date Added:"
+              with:
+          - parseDate: January 2, 2006
+      Details: //div[@class="videoDetails clear"]/p
+      Performers:
+        Name: //li[@class="update_models"]/a
+      Tags:
+        Name: //ul/li[@class="label" and contains(.,"Categories:")]/..//a
+      Studio:
+        Name:
+          selector: //div[@id='logo']/h1/a/span
+        Image:
+          selector: //img[@class="update_thumb thumbs stdimage"]/@src0_1x
+          postProcess:
+            - replace:
+              - regex: .*/tour/content//
+                with: "https://feminized.com/tour/content/"
```

Comment:
the image was shifted a couple of spaces breaking the scraper
and it also need to be adjusted to work with both sites
```yaml
      Image:
        selector: //base/@href|//img[@class="update_thumb thumbs stdimage"]/@src0_1x
        concat: "|"
        postProcess:
          - replace:
              - regex: "(^[^|]+)\\|/tour/([^\\.]+\\.jpg).*"
                with: $1$2
```
seeems to do the job

Reply:
Ah yeah - my bad, I half-baked it.